package.json

```diff
@@ -65,7 +65,7 @@
     "jwt-decode": "^2.2.0",
     "knex": "^2.4.2",
     "lru-cache": "^6.0.0",
-    "moment": "^2.29.4",
+    "moment": "^2.30.1",
```

Comment:
this is what was causing the npm security audit to fail

however, changing this also directly impacts Joi, because Joi uses the Moment library behind the scenes. This then causes a bunch of validators to break - https://github.com/moment/moment/pull/5592 (included in 2.30.1 but not listed as a breaking change!)